cmd/notation/inspect.go

```diff
@@ -103,6 +65,12 @@ Example - Inspect signatures on an OCI artifact identified by a digest and outpu
 			return nil
 		},
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.Format.Parse(cmd); err != nil {
+				return err
+			}
+			if err := opts.Common.Parse(cmd); err != nil {
+				return err
+			}
```

Comment:
You may want https://github.com/oras-project/oras/blob/main/cmd/oras/internal/option/parser.go